helm/grafana/templates/cnpg/objectstores.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if and (.Values.postgresqlCluster.enabled) (.Values.postgresqlCluster.backup.enabled) }}
```

Comment:
Is there a reason why `objectstore-backup` only created if `postgresqlCluster.enabled`, and not `objectstore-recovery`?